samples/Gallery/MainWindow.fs

```diff
@@ -237,14 +237,17 @@ module MainWindow =
             )
 
     let view model =
-        DesktopApplication() {
+        (DesktopApplication() {
             (Window() { hamburgerMenu model })
                 .title("Fabulous Gallery")
                 .menu(createMenu())
                 .width(1024.)
                 .height(800.)
                 .icon("avares://Gallery/Assets/Icons/logo.ico")
-        }
+        })
+#if DEBUG
+            .attachDevTools()
```

Comment:
Does the DevTools work for you ?. I know in Fabulous we need to wrap it properly to make it work

Reply:
Yeah, they open for me when I hit `F12` while running in `Debug` mode.

That reminds me of sth. I ran into on notebook keyboard layouts using the F1-12 keys: Some override them with hardware functions by default and you have to hold down the `fn` key to access the original functionality.

Reply:
Just tried. It works 👍